<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cuteKB</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/ico/table32.png</normaloff>:/images/ico/table32.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="tableView">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWebEngineView" name="textBrowser" native="true">
       <property name="url" stdset="0">
        <url>
         <string>about:blank</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Databas&amp;e</string>
    </property>
    <addaction name="actionNewentry"/>
    <addaction name="actionEditentry"/>
    <addaction name="actionDeleteEntry"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewentry"/>
   <addaction name="actionEditentry"/>
   <addaction name="actionDeleteEntry"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionFindPrevious"/>
   <addaction name="actionFindNext"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Table of contents</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="textBrowserTOC">
       <property name="tabStopWidth" stdset="0">
        <number>81</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNewentry">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/new.png</normaloff>:/images/ico/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEditentry">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/edit.png</normaloff>:/images/ico/edit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDeleteEntry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/remove.png</normaloff>:/images/ico/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Entry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/settings.png</normaloff>:/images/ico/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/arrow-circle-double.png</normaloff>:/images/ico/arrow-circle-double.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFindNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/arrow-curve.png</normaloff>:/images/ico/arrow-curve.png</iconset>
   </property>
   <property name="text">
    <string>FindNext</string>
   </property>
   <property name="toolTip">
    <string>Find next occurance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+.</string>
   </property>
  </action>
  <action name="actionFindPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/arrow-curve-180.png</normaloff>:/images/ico/arrow-curve-180.png</iconset>
   </property>
   <property name="text">
    <string>FindPrevious</string>
   </property>
   <property name="toolTip">
    <string>Find Previous occurance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+,</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/ico/question.png</normaloff>:/images/ico/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About software</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>textBrowserTOC</sender>
   <signal>anchorClicked(QUrl)</signal>
   <receiver>MainWindow</receiver>
   <slot>tocLinkClicked(QUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>1167</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textBrowser</sender>
   <signal>loadFinished(bool)</signal>
   <receiver>actionFindNext</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tocLinkClicked(QUrl)</slot>
 </slots>
</ui>
